### ISSUE TITLE
Is there plan to support Lua Script?

### ISSUE THREAD

**Issue by calvinxiu** [wilkenstein/redis-mock-java#1]
the most difficult task in all mock redis implements are lua support....

**Comment by wilkenstein** [comment]
Yes, there is! All the redis-mock-* projects will eventually support lua scripting.

**Comment by wilkenstein** [comment]
b1183645803153469e2a177872cb35e559e0eba1 Added a LuaScripter that can execute Lua scripts. Now the question is how to replicate redis.call and redis.pcall.... It's interesting, I may work on this earlier than a couple of months!

**Comment by TheHydGuy** [comment]
Do the library support lua scripts?
